File: cellulosesz-api/src/main/java/top/likoslupus/cellulosesz/api/text/TextService.java
package top.likoslupus.cellulosesz.api.text;

import java.util.List;
import java.util.Set;

public interface TextService {

    List<String> info();

    List<String> motd();

    List<String> rules();

    List<String> custom(String name);

    Set<String> customNames();

    int pageSize();

}

File: cellulosesz-api/src/main/kotlin/top/likoslupus/cellulosesz/api/text/TextService.kt
package top.likoslupus.cellulosesz.api.text

public interface TextService {

    public fun info(): List<String>

    public fun motd(): List<String>

    public fun rules(): List<String>

    public fun custom(name: String): List<String>

    public fun customNames(): Set<String>

    public fun pageSize(): Int

}
